editor/plugins/org.fusesource.ide.projecttemplates/.descriptors/blank-spring.xml

```diff
@@ -0,0 +1,17 @@
+<assembly
+	xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3 http://maven.apache.org/xsd/assembly-1.1.3.xsd">
+	<id>blank-spring</id>
+	<includeBaseDirectory>false</includeBaseDirectory>
+	<formats>
+		<format>zip</format>
+	</formats>
+	<fileSets>
+		<fileSet>
+			<directory>${basedir}/../../../templates/blank/blank-spring</directory>
```

Comment:
what do you think of moving the templates folders inside the org.fusesource.ide.projecttemplates plugin, it will avoid to have such relative paths and it will help to gather related things together.
We will just need to exclude the folder in build.properties